src/main/java/fr/insee/pearljam/api/service/impl/UtilsServiceImpl.java

```diff
@@ -7,8 +7,6 @@
 import javax.servlet.http.HttpServletRequest;
```

Comment:
Is this service really needed ? It does complete different things, not sure about this service responsibility.

Ready to discuss about this one if you want :)